src/main/java/net/dv8tion/jda/api/entities/MessageReaction.java

```diff
@@ -399,7 +399,7 @@ public ReactionPaginationAction retrieveUsers()
 
     private String getReactionCode()
     {
-        return emote.isEmote()
+        return emote != null
```

Comment:
When would this ever be null?

Reply:
I don't know either.
`isEmote()` essentially returns the result from `emote != null` and @DManstrator suggested this alteration.

Reply:
No, that makes no sense. You are calling `isEmote` on an object, that object has an `emote` field which might be null.

Reply:
@MinnDevelopment @Andre601 Yeah, it was my bad, I got confused here. I saw that in the `ReactionEmote` class `emote` actually can be null in case of an emoji.
https://github.com/DV8FromTheWorld/JDA/blob/5234d2d943342550a9fe6605e78463f25e4589ff/src/main/java/net/dv8tion/jda/api/entities/MessageReaction.java#L437-L443
The variables just had the same name and the method was close to the `ReactionEmote` class, that's why I thought that it is the nullable `emote`. Again sorry for that.

Reply:
So, should I revert it back to emote.isEmote()?